### ISSUE TITLE
Execution tracing

### ISSUE THREAD

**Issue by alcuadrado** [ethereum/evmc#534]
Hi,

I'm exploring the possibility of using a native implementation of an EVM and its host in [Buidler](https://buidler.dev/) to run smart contract tests. We currently use `ethereumjs-vm`, so I believe this will lead to really significant performance improvements.

Some of the main features Buidler offers are based on tracing the smart contracts' execution in order to infer and present meaningful information to our users, instead of just directly exposing the EVM-level results and errors.

I noticed that the tracing functionality was dropped from EVMC, so I'm opening this issue to ask if there are any plans to replace it with a new one.

**Comment by chfast** [comment]
Yes, there are plans to re-add it, but it had low priority. I wanted to do it after https://github.com/ethereum/evmc/issues/44. Let me know if you still need it so I can adjust my plans.

**Comment by alcuadrado** [comment]
Thanks for the reply @chfast. I have discarded this project, at least for now, to focus on other Buidler stuff. Thanks anwyay.

**Comment by axic** [comment]
An update on this: evmone has some support for tracing merged, though it is not part of the EVMC interface.

Also we are considering an Wasm version (including for use with etheruemjs-vm), which can be tracked in https://github.com/ethereum/evmone/issues/293
